test/unit/uk/gov/hmrc/thirdpartyapplication/controllers/AuthorisationWrapperSpec.scala

```diff
@@ -66,49 +60,49 @@ class AuthorisationWrapperSpec extends UnitSpec with MockitoSugar with WithFakeA
 
     "accept the request when access type in the payload is PRIVILEGED and gatekeeper is logged in" in new Setup {
 
-      mockAuthConnectorToReturn(true)
+      givenUserIsAuthenticated(underTest)
 
-      val result = await(underTest.requiresRoleFor(APIGatekeeper, PRIVILEGED).async(BodyParsers.parse.json)(_ =>
+      val result = await(underTest.requiresRoleFor(PRIVILEGED).async(BodyParsers.parse.json)(_ =>
         Default.Ok(""))(privilegedRequest)
       )
 
       status(result) shouldBe SC_OK
     }
 
-    "accept the request when access type in the payload is ROPC and gatekeeper is logged in" in new Setup {
-      mockAuthConnectorToReturn(true)
-      status(underTest.requiresRoleFor(APIGatekeeper, ROPC).async(BodyParsers.parse.json)(_ => Default.Ok(""))(ropcRequest)) shouldBe SC_OK
+    "accept the request when access type in the payload is ROPC and user is authenticated" in new Setup {
+      givenUserIsAuthenticated(underTest)
+      status(underTest.requiresRoleFor(ROPC).async(BodyParsers.parse.json)(_ => Default.Ok(""))(ropcRequest)) shouldBe SC_OK
     }
 
     "skip gatekeeper authentication for payload with STANDARD applications" in new Setup {
 
-      val result = await(underTest.requiresRoleFor(APIGatekeeper, PRIVILEGED).async(BodyParsers.parse.json)(_ =>
+      val result = await(underTest.requiresRoleFor(PRIVILEGED).async(BodyParsers.parse.json)(_ =>
```

Comment:
Says "STANDARD" in test name but uses "PRIVILEGED" in test code. Is that right?

Reply:
What does this test mean?

Reply:
It uses a standardRequest and requires authorisation if it were privileged, therefore it doesn't call need to make a call to auth. Have renamed for clarity, hopefully.